docs/getting-started/summaryofchanges.md

```diff
@@ -52,26 +52,25 @@ The following features and enhancements were added.
+- In this version, cross-launch via URL has been implemented, allowing for integration between Zlux and other web applications. This feature enables users to bookmark a set of app2app communication actions (in the form of a URL) that will be executed when opening the webpage [#234](https://github.com/zowe/zlux-app-manager/pull/234)
+- Improvements have been made to the bookmarking functionality in the TN3270 emulator [#30](https://github.com/zowe/tn3270-ng2/pull/30)
+  - Codepages have been reorganized so that the numbers are shown first, making it easier for users to navigate to their favorites
+  - Users can now save connection preferences on a per-user level. Clicking the floppy disk icon saves user settings to that user's scope.
+  - The buttons found in this feature have been realigned  
+- Several features have been added to the Zowe Editor [#153](https://github.com/zowe/zlux-editor/pull/153)
   - Globally increased the shortest duration of snackbar notifications from 2 seconds to 3 seconds
   - Added a "Close All" button in the menu (hotkey is Alt + W + Shift)
-  - Added a snackbar notification for opening a folder/file you don't have access to
+  - A snackbar notification will be displayed when users attempt to open a file that they do not have permission to open
   - Added an "Undo" option to the Close All feature to re-open tabs & files
-- Login activity and storage event updates [#242](https://github.com/zowe/zlux-app-manager/pull/242)
-  - Converted login activity to event emitter
-  - Added new local storage listener so changes can be reported across tabs
-  - Added detect activity when loging in from a new window/tab
-  - In idleWarn popup added
-  - Being active on the Desktop, while the session expiration prompt is active, will now dismiss the pop-up and attempt to renew the session. This feature also stays synchronized between multiple tabs, and will renew multiple tabs if the user is active in at least one of them
-- NGX-monaco has been removed [#155](https://github.com/zowe/zlux-editor/pull/155)
+- Login activity and session activity is now synchronized across multiple desktop tabs [#242](https://github.com/zowe/zlux-app-manager/pull/242)
+  - When a user logs out of a desktop tab, all other active tabs will also log out
+  - When a user performs an action on a desktop tab, the other tabs register this activity, which stops them from timing out
+- NGX-monaco-editor library has been removed, allowing the Editor to open and view files after the second instance of opening them [#155](https://github.com/zowe/zlux-editor/pull/155)
   - Removed use of node-sass, so that native compilation is not required
-  - Updated to typescript 3.7
-  - Updated to monaco 0.20
+  - Updated to typescript 3.7 from version 2.7.2
+  - Updated to monaco 0.2 from version 0.13. The monaco changelog can be found [here](https://github.com/microsoft/monaco-editor/blob/master/CHANGELOG.md)
```

Comment:
I believe 0.2 exists and is not the same as 0.20, please keep it 0.20 for correctness.